addons/mail/static/src/xml/abstract_thread_window.xml

```diff
@@ -17,7 +17,13 @@
             <div class="o_thread_window_content">
             </div>
             <div t-if="widget.needsComposer()" class="o_thread_composer o_chat_mini_composer">
-                <input class="o_composer_text_field" t-att-placeholder="widget.options.placeholder"/>
+                <div class="o_composer_container">
+                    <div class="o_composer">
+                        <div class="o_composer_input o_customer_composer">
+                            <textarea class="o_composer_text_field" t-att-placeholder="widget.options.placeholder"/>
+                        </div>
+                    </div>
+                </div>
```

Comment:
- Seems strange that the AbstractThreadWindow template would need the thread_window.scss file to work and not only the abstract_thread_window.scss file, no?
- What is the purpose of the new o_customer_composer class ?

Reply:
- Indeed, it's strange. I will check, but the discuss popup is different from livechat's one. I think it uses thread_window and livechat uses abstract_thread_window (maybe I am wrong).

- The discuss composer is divided in two parts, the textarea and smiley/attachment buttons. As livechat doesn't provide theses features, the .o_customer_composer set the textarea container to full width

I had doubts that it would pass in stable. I found a way to stylize only the livechat but I knew that the problem was deeper. I will make another PR with only CSS for stable and retarget this one for master